app/javascript/user_mentor_memo.js

```diff
@@ -0,0 +1,16 @@
+import Vue from 'vue'
+import UserMentorMemo from './user_mentor_memo.vue'
+
+document.addEventListener('DOMContentLoaded', () => {
+  const userMentorMemo = document.querySelector('#js-user-mentor-memo')
```

Comment:
```suggestion
  const selector = '#js-user-mentor-memo'
  const userMentorMemo = document.querySelector(selector)
```

みたいにすると下の方と共通化できていいかもです〜

Reply:
@komagata 
ご確認ありがとうございます！
f0d031b にて対応いたしました！